File: code-examples/java/users/delete.java
HttpResponse<String> response = Unirest.delete("http://api.marketcloud.it/v0/users/3")
  .header("accept", "application/json")
  .header("content-type", "application/json")
  .header("authorization", "YourPublicKey:YourToken")
  .asString();

File: code-examples/python/users/delete.py
import http.client

conn = http.client.HTTPConnection("api.marketcloud.it")

headers = {
    'accept': "application/json",
    'content-type': "application/json",
    'authorization': "YourPublicKey:YourToken"
    }

conn.request("DELETE", "/v0/users/3", headers=headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))